test/easy_localization_widget_test.dart

```diff
@@ -105,12 +105,8 @@ void main() async {
         expect(pluralFinder, findsOneWidget);
 
         expect(tr('test'), 'test');
-        expect(plural('day', 1), '1 day');
```

Comment:
Why are those expects removed? In case they tested the wrong behaviour, couldnt we update the expected value here instead?

Reply:
There's tests that test this logic specifically. It's generally a good idea to keep unit tests to a single tested thing per test, so that - for example - all tests don't have to be rewritten if the expected logic or API changes: just the one specific test needs to be updated.